drivers/ethernet/CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ if(CONFIG_ETH_NXP_S32_NETC)
   zephyr_library_sources_ifdef(CONFIG_DT_HAS_NXP_S32_NETC_VSI_ENABLED	eth_nxp_s32_netc_vsi.c)
 endif()
 
+if(CONFIG_ETH_NXP_IMX_NETC)
+  zephyr_library_sources(eth_nxp_imx_netc.c)
+  zephyr_library_sources_ifdef(CONFIG_DT_HAS_NXP_IMX_NETC_PSI_ENABLED	eth_nxp_imx_netc_psi.c)
```

Comment:
the `_ifdef` here also seems redundant since CONFIG_ETH_NXP_IMX_NETC depends on CONFIG_DT_HAS_NXP_IMX_NETC_PSI_ENABLED , just add both files as sources (which btw, why is this driver in two files?)

Reply:
You are right. We should add both here. I will fix.
The reason we have two files here, is because one is netc common driver, and one is netc psi driver.
We will also add netc vsi driver in the future.

Thanks.